NMLocalizedPhoneCountryView/NMLocalizedPhoneCountryView.swift

```diff
@@ -418,6 +418,16 @@ extension NMLocalizedPhoneCountryView: NMLocalizedPhoneCountryViewDataSource {
         return dataSource?.navigationTitle(in: self)
     }
     
+    var searchBarPlaceHolderTitle: String? {
+        
+        return dataSource?.searchBarPlaceholderTitle(in: self)
+    }
+    
+    var searchBarCancelButtonTitle: String? {
+        
+        return dataSource?.searchBarCancelButtonTitle(in: self)
```

Comment:
For search button and placeholder titles, provide a default one (like done for UIBarButton close) if not provided

Reply:
Done